### DISCUSSION TITLE
`--open` flag for `next dev` to automatically open in a browser

### DISCUSSION THREAD

**Discussion by gleich** [vercel/next.js#24439] [Ideas]
### Describe the feature you'd like to request

It would be great to add a flag to the `next dev` command to have the localhost port automatically opened in the user's default browser. I would love to make a PR for this!

### Describe the solution you'd like

Sindre Sorhus's package [open](https://github.com/sindresorhus/open) seems like it would get the job done.

### Describe alternatives you've considered

I've been running `next dev && open localhost:3000` as an alternative but it isn't a cross-platform solution.

**Comment by darshkpatel** [comment]
+1 for the Feature
I'd love to help get this integrated if the devs give a green flag

**Comment by gleich** [comment]
I should be all set but I'll definitely hit ya up @darshkpatel if I need any help :)

**Comment by AnweshGangula** [comment]
This was already discussed previously - #13448 & #12233 

> We're not planning to add automatically opening the browser at this point (keeping the CLI minimal and features contained).
